src/table/components/head/HeadCellMenu.tsx

```diff
@@ -72,19 +72,31 @@ export default function HeadCellMenu({
 
   return (
     <>
-      <StyledMenuIconButton
-        isVisible={openListboxDropdown || openMenuDropdown}
-        ref={anchorRef}
-        size="small"
-        tabIndex={-1}
-        id="sn-table-head-menu-button"
-        aria-controls={openMenuDropdown ? 'sn-table-head-menu' : undefined}
-        aria-expanded={openMenuDropdown ? 'true' : undefined}
-        aria-haspopup="true"
-        onClick={() => setOpenMenuDropdown(!openMenuDropdown)}
-      >
-        <More size="small" />
-      </StyledMenuIconButton>
+      <div>
+        <StyledMenuIconButton
+          isVisible={openListboxDropdown || openMenuDropdown}
+          ref={anchorRef}
+          size="small"
+          tabIndex={-1}
+          id="sn-table-head-menu-button"
+          aria-controls={openMenuDropdown ? 'sn-table-head-menu' : undefined}
+          aria-expanded={openMenuDropdown ? 'true' : undefined}
+          aria-haspopup="true"
+          onClick={() => setOpenMenuDropdown(!openMenuDropdown)}
+        >
+          <More size="small" />
+        </StyledMenuIconButton>
+        {openListboxDropdown && (
```

Comment:
how come this is inside a new div next to the button?

Reply:
we have a new PR related to the popover version of listbox. 
this one is closed since the solution in the nebula got changed.